base/tuple.jl

```diff
@@ -96,6 +96,11 @@ function indexed_iterate(I, i, state)
     x
 end
 
+rest(t::Tuple) = t
+rest(t::NTuple{N}, i::Int) where {N} = ntuple(x -> getfield(t, x+i-1), N-i+1)
```

Comment:
```suggestion
rest(t::Tuple, i::Int) = ntuple(x -> getfield(t, x+i-1), length(t)-i+1)
```

Reply:
Oh, that's a good point! Didn't realize that:
```
julia> (1, 1.) isa NTuple{2}
false
```